docs/arch/lexicon.md

```diff
@@ -0,0 +1,9 @@
+
+## Inter-Service Lexicon
+
+- **Action**: a CI atomic unit. It contains infrastructure, environment and commands to execute.
+- **Agent**: an agent is a computing node registered with the scheduler.
+- **Agent pool**: the set of all registered agents. It is a scheduler's entire knowledge about available computing resources.
+- **Scheduling**: selection of an agent on which to execute an action.
+- **Action status**: the state of the execution of an action (running, successful, failed).
+- **Pipeline**: a set of actions to be executed. It is declared as a YAML file.
```

Comment:
Please order those terms alphabetically and we should be good.

Reply:
Done!